x-pack/plugins/security_solution/public/detection_engine/rule_management/api/api.ts

```diff
@@ -194,6 +195,7 @@ export interface BulkActionResult {
   updated: Rule[];
   created: Rule[];
   deleted: Rule[];
+  skipped?: Rule[];
```

Comment:
It shouldn't be optional for consistency. Basically it's enough to return an empty array.